### PR TITLE
Add translate for SPZ

### PR DESCRIPTION
**Pull request by lucaferranti**
- add minkowski sum for sparse polynomial zonotope
- add translate for spz

### CODE REVIEW

File: src/Sets/SparsePolynomialZonotope.jl [hunk 1]
```diff
@@ -318,6 +318,47 @@ end
...
+
+"""
+    traslate(S::SparsePolynomialZonotope, v::AbstractVector)
+
+Translate (i.e., shift) a sparse polynomial zonotope by a given vector.
+
+### Input
+
+- `S` -- sparse polynomial zonotope
+- `v` -- transaltion vector
```
Comment (schillic) [inline src/Sets/SparsePolynomialZonotope.jl:351]:
```suggestion
- `v` -- translation vector
```

File: docs/src/lib/sets/SparsePolynomialZonotope.md [hunk 2]
```diff
@@ -20,5 +20,7 @@ nparams(::SparsePolynomialZonotope)
 order(::SparsePolynomialZonotope)
 linear_map(::Union{Real, AbstractMatrix, LinearAlgebra.UniformScaling}, ::SparsePolynomialZonotope)
 exact_sum(::SparsePolynomialZonotope, ::SparsePolynomialZonotope)
+minkowski_sum(::SparsePolynomialZonotope, ::SparsePolynomialZonotope)
+translate(S::SparsePolynomialZonotope, v::AbstractVector)
```
Comment (schillic) [inline docs/src/lib/sets/SparsePolynomialZonotope.md:24]:
```suggestion
translate(::SparsePolynomialZonotope, ::AbstractVector)
```

**Review by schillic — APPROVED** [review]

**Review by mforets — APPROVED** [review]
It is sometimes useful to have the in-place version as well, but we add that feature later (still missing a few methods in https://github.com/JuliaReach/LazySets.jl/pull/1995 )

**Comment by mforets** [comment]
@lucaferranti conflict ^

**Comment by lucaferranti** [comment]
@mforets should be ready to go now ^

**Pull request merged**